doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/configs/migration-example.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: raycluster-example
+spec:
+  # To use autoscaling, the following field must be included.
+  enableInTreeAutoscaling: true
+  # The Ray version must be supplied.
+  rayVersion: '2.0.0'
+  headGroupSpec:
+    serviceType: ClusterIP
+    rayStartParams:
+      dashboard-host: '0.0.0.0'
+      block: 'true'
+      # Annotate the head pod as having 0 CPU
+      # to prevent the head pod from scheduling Ray workloads.
+      num-cpus: 0
+    template:
+      spec:
+        containers:
+        - name: ray-head
+          image: rayproject/ray-ml:2.0.0-gpu
+          resources:
+            limits:
+              cpu: "14"
+              memory: "54Gi"
+            requests:
+              cpu: "14"
+              memory: "54Gi"
+          # Keep this in container configs.
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh","-c","ray stop"]
+  workerGroupSpecs:
+  # Start with 2 CPU workers. Allow scaling up to 3 CPU workers.
+  - replicas: 2
+    minReplicas: 2
+    maxReplicas: 3
+    groupName: rayCPUWorkerType
+    rayStartParams:
+      block: 'true'
+      # Annotate the Ray worker pod as having 1 unit of "Custom" capacity and 5 units of "Custom2" capacity
+      resources: '"{\"Custom\": 1, \"Custom2\": 5}"'
+    template:
+      spec:
+        containers:
+        - name: ray-worker
+          image: rayproject/ray-ml:2.0.0-gpu
+          resources:
+            limits:
+              cpu: "14"
+              memory: "54Gi"
+            requests:
+              cpu: "14"
+              memory: "54Gi"
+          # Keep the lifecycle block in Ray container configs.
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh","-c","ray stop"]
+        # Keep the initContainers block in worker pod configs.
```

Comment:
Hmm, why are these initContainers needed? I haven't seen them anywhere else :)

Reply:
Yep, they're in every sample config. Definitely something to sort out.